### PR TITLE
java 11 compatibility: handle a new exception if a field is inaccessible

### PR DESCRIPTION
**Pull request by mprusakov**

### CODE REVIEW

File: src/main/java/org/ehcache/sizeof/ObjectGraphWalker.java [hunk 1]
```diff
@@ -263,6 +263,9 @@ private static void nullSafeAdd(final Deque<Object> toVisit, final Object o) {
                         LOG.error("Security settings prevent Ehcache from accessing the subgraph beneath '{}'" +
                                   " - cache sizes may be underestimated as a result", field, e);
                         continue;
+                    } catch (RuntimeException e) {
+                        LOG.warn("The JVM is preventing Ehcache from accessing the subgraph beneath '{}'" +
+                                " - cache sizes may be underestimated as a result", field, e);
```
Comment (chrisdennis) [inline src/main/java/org/ehcache/sizeof/ObjectGraphWalker.java:268]: Missing the `continue`... probably missing from my original example code.

Comment (mprusakov) [reply src/main/java/org/ehcache/sizeof/ObjectGraphWalker.java:268]: fixed

**Review by chrisdennis — APPROVED** [review]

**Comment by mprusakov** [comment]
Many thanks for that, would it be possible to release this change (possibly along with https://github.com/ehcache/sizeof/pull/60) as there are no java 11 compatible releases yet.

**Comment by chrisdennis** [comment]
Yes. That should happen either later today or early next week.

**Pull request merged**